apps/mudita-center-e2e/src/page-objects/mc-update-modal.page.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import Page from "./page"
+
+class MCModalPage extends Page {
```

Comment:
🟢 
```suggestion
class ModalPage extends Page {
```
or (if needed)
```suggestion
class ModalPage extends BasePage {
```

Would you consider avoiding the "MC" prefix in class names? Instead, adding the prefix to the base class or interface could maintain clarity and consistency in naming, allowing for more concise derived class names and reducing nested redundancy. What are your thoughts on this approach?